src/chanprog.c

```diff
@@ -366,6 +367,12 @@ void tell_verbose_status(int idx)
                "Threaded DNS core is disabled.\n"
 #endif
                "Socket table: %d/%d\n", threaddata()->MAXSOCKS, max_socks);
+
+  if (!getrlimit(RLIMIT_NOFILE, &rlp)) /* TODO: should be posix 2001
+                                        * make sure this is as portable as expected
+                                        * and maybe also add reporting of
+                                        * current amount of open files */
+    dprintf(idx, "Maximum number of open files (sockets): cur %ju max %ju\n", rlp.rlim_cur, rlp.rlim_max);
```

Comment:
This does need casts to (uint_max) because variadic functions don't auto-cast and the size they fetch depends on the specifier